src/resources/dev-envs/clion.md

```diff
@@ -0,0 +1,80 @@
+# CLion
+
+CLion is a commercial IDE from JetBrains. Altough it is expensive for regular users, it is free for university students. <br>
+You can check if you school is on the list in the swot repo: <https://github.com/JetBrains/swot>
+
+## Applying for a license
+
+You can apply for a student license at: <https://www.jetbrains.com/shop/eform/students> <br>
+Use your university email address. After you complete the application form you will be prompted to create a JetBrains account. Your license will be tied to this account. You will recieve a confirmation email about your license application to your university email address.
+
+## Downloading CLion
+
+You can download CLion at: <https://www.jetbrains.com/clion/download>
+
+
+During installation, tick the "Add bin folder to PATH" and "Add Open Folder as Project" options.
+
+After completing the installation, you should be prompted to log into your JetBrains account.<br>
+If this doesn't happen, you can log in from the menu later after creating a project: Help -> Register...
+
+## Installing devtools
+
+CLion ships with MinGW GCC on Windows. If you want to install MSVC, use the Visual Studio Installer to install it. You will be able to use MSVC from CLion.
+
+## Creating a new project
+
+After you installed and activated CLion with your JetBrains account, you should see this page:
+
+![alt text](../../public/assets/clion/clion-1.png)
```

Comment:
Do not use relative paths. Use absolute paths from root. See the Visual Studio guide for how this works.